app/swagger/swagger/requests/form1095_bs.rb

```diff
@@ -22,7 +22,7 @@ class Form1095Bs
               property :available_forms, type: :array do
                 items do
                   property :year, type: :integer, example: 2021
-                  property :last_updated, type: :string, example: '2022-08-03T16:08:50.071Z'
+                  property :last_updated, type: %i[string null], example: '2022-08-03T16:08:50.071Z'
```

Comment:
This property is not used by the front end and will be removed entirely in the near future.